MidtransKit/MidtransKit/VTErrorStatusController.m

```diff
@@ -7,13 +7,22 @@
 //
 
 #import "VTErrorStatusController.h"
+#import "VTClassHelper.h"
 
 @interface VTErrorStatusController ()
-
+@property (nonatomic) NSError *error;
 @end
 
 @implementation VTErrorStatusController
 
+- (instancetype)initWithError:(NSError *)error {
+    UIStoryboard *storybaord = [UIStoryboard storyboardWithName:@"Midtrans" bundle:VTBundle];
+    if (self = [storybaord instantiateViewControllerWithIdentifier:@"VTErrorStatusController"]) {
+        self.error = error;
+    }
+    return self;
+}
+
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
```

Comment:
ah got it, do you think it's important to remove it, because it's auto generated comment

@vanbungkring